genetic_algorithm/basic_string.py

```diff
@@ -0,0 +1,148 @@
+"""
+Simple multithreaded algorithm to show how the 4 phases of a genetic
+algorithm works (Evaluation, Selection, Crossover and Mutation)
+
+https://en.wikipedia.org/wiki/Genetic_algorithm
+
+
+Author: D4rkia
+"""
+
+from typing import List, Tuple
+import random
+
+N_POPULATION = 200
+N_SELECTED = 50
+MUTATION_PROBABILITY = 0.1
+random.seed(random.randint(0, 1000))
```

Comment:
Please add a comment on this line or just above to explain why this command is useful.

Reply:
Fixed